config/locales/it.yml

```diff
@@ -578,8 +579,46 @@ it:
+      licensetext: "<div class=\"box wide\">Tutte le azioni sul sito sono pubblice ad eccezione dei messaggi privati tra utenti. Le azioni di moderazione possono essere consultate sul <a href=\"/moderations\">Log di Moderazione</a>.<br/><a rel=\"license\" href=\"https://creativecommons.org/licenses/by/4.0/deed.it\"><img alt=\"Licenza Creative Commons\" style=\"border-width:0\" src=\"https://i.creativecommons.org/l/by/4.0/88x31.png\" /></a><br /><span xmlns:dct=\"http://purl.org/dc/terms/\" property=\"dct:title\">Gambe.ro</span> è pubblicato secondo i termini della <a rel=\"license\" href=\"https://creativecommons.org/licenses/by/4.0/deed.it\">licenza Creative Commons Attribution 4.0 International</a>.</div>"
```

Comment:
pubbliche*

"Log di Moderazione" lo metterei minuscolo, essendo in mezzo a una frase.

Reply:
Corretto pubbliche. Log di Moderazione è un nome proprio, un po' tipo "Timeline" su Facebook o Feed su Twitter. Nelle altre versioni sono in maiuscolo.